examples/multi_tunnels/prod.tf

```diff
@@ -49,6 +49,7 @@ module "vpn-gw-us-ce1-prd-mgt-internal-01" {
   peer_ips           = [module.vpn-gw-us-ce1-mgt-prd-internal-01.gateway_ip, module.vpn-gw-us-ce1-mgt-prd-internal-02.gateway_ip]
 
   cr_name                  = "cr-uscentral1-to-mgt-vpc-tunnels-01"
+  cr_enabled = true
```

Comment:
Doesn't brake formatting, but aligned might look better

Reply:
how come terraform fmt passed that?

Reply:
@kopachevsky, 1 white space is ok for it. 
So it passes the check, but still produces some level of distraction if the styles are mixed in 1 file.. I'm ok with the other way, but we may want to auto-format everything that way.